samples/bluetooth/mesh/light/boards/thingy53_nrf5340_cpuapp.conf

```diff
@@ -13,8 +13,15 @@ CONFIG_MCUMGR_SMP_BT_AUTHEN=n
 CONFIG_MCUMGR_CMD_IMG_MGMT=y
 CONFIG_MCUMGR_CMD_OS_MGMT=y
 
-CONFIG_BT_BUF_ACL_RX_SIZE=264
-CONFIG_BT_L2CAP_TX_MTU=260
+CONFIG_BT_L2CAP_TX_MTU=247
+CONFIG_BT_BUF_ACL_TX_SIZE=251
+CONFIG_BT_BUF_ACL_RX_SIZE=251
+
+# Use connection parameters that ensure quick data transfer
+CONFIG_BT_PERIPHERAL_PREF_MIN_INT=6
```

Comment:
Instead of setting the PPCP globally for faster DFU it would be better to change the connection parameters during DFU.
In the DFU service we could request a change of connection parameters.

In general setting the lowest connection parameter is not really the optimal in terms of bandwidth, instead it improves the latency.
It looks like you can still speed this up by increasing buffer count or possibly CPU processing, because it seems you are not able to keep the connection event alive by having the buffers full.
It could also be a limitation in the connection event length, I suggest looking into if this can be increased.

Reply:
On Android there is a way to set connection interval to 11.25ms, on iOS there's not.
Device could requires fast parameters when in image managern the upload command is called with off=0 and switch back to when upload complete. As Marek says, that's additional logic.

Reply:
I think we could use slave latency = 0 set in Kconfig for now and then add the logic (maybe even to Zephyr's SMP BT module). I would prefer to avoid introducing new functionalities at the moment.

Reply:
@MarekPieta That's acceptable as a temporary solution, but please make sure to have tracking issue for that.

Reply:
Ticket for that one is already added :)

Reply:
@MarekPieta please add the ticket id here for reference.

Reply:
Jira ticket id: NCSDK-11180